icu4c/source/common/unicode/urename.h

```diff
@@ -1188,42 +1175,6 @@
 #define ulocdata_getPaperSize U_ICU_ENTRY_POINT_RENAME(ulocdata_getPaperSize)
 #define ulocdata_open U_ICU_ENTRY_POINT_RENAME(ulocdata_open)
 #define ulocdata_setNoSubstitute U_ICU_ENTRY_POINT_RENAME(ulocdata_setNoSubstitute)
-#define ulocimp_addLikelySubtags U_ICU_ENTRY_POINT_RENAME(ulocimp_addLikelySubtags)
```

Comment:
@roubert has done a lot of refactoring in the locale ID code. It's possible that some or many of these are either gone or are now in an anonymous namespace. But there should also be some new ones.